frb/scripts/pz_dm.py

```diff
@@ -61,10 +62,19 @@ def main(pargs):
 
     z_min = z[np.argmin(np.abs(cum_sum-limits[0]/100.))]
     z_max = z[np.argmin(np.abs(cum_sum-limits[1]/100.))]
+    z_50 = z[np.argmin(np.abs(cum_sum-50./100.))]
 
     # Finish
     print("")
+    print(f"Allowing for the MW and Host, DM_cosmic = {int(DM_cosmic)} pc/cm^3")
+    print("")
+    print("")
+    print(f"The mean redshift value is: {z_50:.3f}")
```

Comment:
Would it be better to report a median/mode of the redshift as the PDF is not symmetric?

Reply:
This just a minor curiosity otherwise looks good! :)

Reply:
mode now reported